wiki.md

```diff
@@ -1,4 +1,4 @@
-v0.7.2
+v088.0
```

Comment:
?

Reply:
First had a type in the version and then generated the wiki.